src/components/Tag/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Pressable from '../Pressable';
+
+export interface ITag extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'size'> {
+  label: string;
```

Comment:
Wouldn't be better to use children instead of label prop ? with children you can also pass other components, might be the use case sometime.

Reply:
We have "leftContent" and "rightContent" for this purpose, ideally a Tag/Chip will be limited to text and icon. Also there is no option here to have an "icon only" tag or chip